sample/F8/speakers

```diff
@@ -0,0 +1,1552 @@
+[
+   {
+      "id":2,
+      "name":"Michael Slater",
+      "email":"",
+      "mobile":"",
+      "photo":"",
+      "organisation":"Facebook",
+      "sessions":[
+         {
+            "id":5,
+            "title":"Introduction to the Camera Effects Platform"
+         }
+      ],
+      "country":"",
+      "short_biography":"",
+      "website":"",
+      "twitter":"",
+      "facebook":"",
+      "github":"",
+      "linkedin":"",
+      "position":"Product Manager",
+      "city":"",
+      "gender":"male",
+      "heard_from":"",
+      "icon":"",
+      "small":"",
+      "speaking_experience":"",
+      "sponsorship_required":"",
+      "thumbnail":"",
+      "featured":true
+   },
+   {
+      "id":4,
+      "name":"Lucas Rocha ",
+      "email":"",
+      "mobile":"",
+      "photo":"",
+      "organisation":"Facebook",
+      "sessions":[
+         {
+            "id":7,
+            "title":"Litho: A Declarative Framework for Efficient UIs"
+         }
+      ],
+      "country":"",
+      "short_biography":"",
+      "website":"",
+      "twitter":"",
+      "facebook":"",
+      "github":"",
+      "linkedin":"",
+      "featured":false,
+      "position":"Engineering Manager",
+      "city":"",
+      "gender":"male",
+      "heard_from":"",
+      "icon":"",
+      "small":"",
+      "speaking_experience":"",
+      "sponsorship_required":"",
+      "thumbnail":""
+   },
+   {
+      "id":5,
+      "name":"Emil Sjölander",
+      "email":"",
+      "mobile":"",
+      "photo":"",
+      "organisation":"Facebook",
+      "sessions":[
+         {
+            "id":7,
+            "title":"Litho: A Declarative Framework for Efficient UIs"
+         }
+      ],
+      "country":"",
+      "short_biography":"",
+      "website":"",
+      "twitter":"",
+      "facebook":"",
+      "github":"",
+      "linkedin":"",
+      "featured":false,
+      "position":"Software Engineer",
+      "city":"",
+      "gender":"male",
+      "heard_from":"",
+      "icon":"",
+      "small":"",
+      "speaking_experience":"",
+      "sponsorship_required":"",
+      "thumbnail":""
+   },
+   {
+      "id":6,
+      "name":"Justin Rogers",
+      "email":"",
+      "mobile":"",
+      "photo":"",
+      "organisation":"Facebook",
+      "sessions":[
+         {
+            "id":8,
+            "title":"WebVR: Enabling Developers to Build Engaging Cross-Platform VR Content"
+         }
+      ],
+      "country":"",
+      "short_biography":"",
+      "website":"",
+      "twitter":"",
+      "facebook":"",
+      "github":"",
+      "linkedin":"",
+      "featured":false,
```

Comment:
remember to add atleast 10-15 featured speakers. do that by making "featured" field as true.